src/tiles/list/tile-list.ts

```diff
@@ -0,0 +1,326 @@
+/* eslint-disable lit/no-invalid-html */
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
+import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
+
+@customElement('tile-list')
+export class TileList extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
+  render() {
+    return html`
+      <div id="list-line" class="${this.classSize}">
+        <div id="list-line-left">
+          <div id="thumb">${this.img()}</div>
+          ${this.iconLeft()}
+        </div>
+        <div id="list-line-right">
+          <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+          ${this.itemLine()} ${this.creator()}
+          <div id="date">
+            <span class="label">Published:</span> ${formatDate(
+              this.date,
+              this.formatSize
+            )}
+          </div>
+          <div id="views-line">
+            ${this.views()} ${this.rating()} ${this.reviews()}
+          </div>
+          ${this.topics()} ${this.description()}
+        </div>
+      </div>
+    `;
+  }
+
+  private img() {
+    if (!this.model?.identifier) {
+      return html``;
+    }
+    return html` <img
+      src="${this.baseNavigationUrl}/services/img/${this.model.identifier}"
+      alt="${this.model.identifier}"
+      class=${this.model?.mediatype}
+    />`;
+  }
+
+  private iconLeft() {
+    if (this.classSize !== 'desktop') {
+      return html``;
+    }
+    return html`
+      <div id="icon-left">
+        <mediatype-icon .mediatype=${this.model?.mediatype} .showText=${true}>
+        </mediatype-icon>
+      </div>
+    `;
+  }
+
+  private creator() {
```

Comment:
Suggestion: use getter syntax for all of these templates and suffix the names with `Template`